### PR TITLE
Role/build infra

### PR DESCRIPTION
**Pull request by Polina-Gubina**

### CODE REVIEW

File: build_infrastructure/molecule/default/files/infrastructure/settings.tf [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    opentelekomcloud = ">= 1.15.0"
+  }
+}
+
+# Configure the OpenTelekomCloud Provider
+provider "opentelekomcloud" {
+  domain_name = "OTC00000000001000000448"
+  tenant_name = "eu-de_csm"
+  auth_url = "https://iam.eu-de.otc.t-systems.com/v3"
+  #auth_url  = "https://iam.eu-de.otc.t-systems.com:443/v3"
```
Comment (lego963) [inline build_infrastructure/molecule/default/files/infrastructure/settings.tf]: remove excess line and add endline

File: build_infrastructure/files/get_key.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        _generate_pub_for_private(private_key_file)
+        print('Private key downloaded')
+    else:
+        _generate_pub_for_private(private_key_file)
+        print('Private key is up to date')
+    return private_key_file
+
+
+if __name__ == '__main__':
+    os.chmod(get_key_from_s3(), RW_OWNER)
```
Comment (outcatcher) [inline build_infrastructure/files/get_key.py]: Please add newline

File: build_infrastructure/molecule/default/files/infrastructure/variables.tf [hunk 3]
```diff
@@ -0,0 +1,4 @@
+variable "username" {}
+variable "password" {}
+
+
```
Comment (outcatcher) [inline build_infrastructure/molecule/default/files/infrastructure/variables.tf]: Please remove empty lines

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: build_infrastructure/molecule/default/files/infrastructure/variables.tf (hunk 3 above)
Comment (anton-sidelnikov) [inline build_infrastructure/molecule/default/files/infrastructure/variables.tf]: remove endline

File: .gitignore [hunk 4]
```diff
@@ -2,3 +2,4 @@
 ### Ansible template
 *.retry
 .idea/
+.terraform/
```
Comment (lego963) [inline .gitignore]: endline pls

File: build_infrastructure/defaults/main.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  TF_VAR_cloud: "otc"
+  TF_VAR_region: "eu-de"
+  TF_VAR_availability_zone: "eu-de-03"
+  TF_VAR_ecs_flavor: "s2.medium.2"
+  TF_VAR_ecs_image: "Debian_10_latest"
+  TF_VAR_addr_3_octets: "192.168.0"
+  TF_VAR_domain_name: "OTC00000000001000000448"
+
+
+tf_variables:
```
Comment (lego963) [inline build_infrastructure/defaults/main.yml:22]: remove if you don't use

File: build_infrastructure/defaults/main.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+tmp_dir: '/tmp'
+key_name: null
+key_path: "{{ tmp_dir }}/{{ key_name }}"
+infrastructure_dir: "infrastructure"
```
Comment (anton-sidelnikov) [inline build_infrastructure/defaults/main.yml:27]: add terraform version 0.12.20

File: register_host/molecule/default/files/scenarios/infrastructure/infrastructure.tf [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  }
+}
+
+variable "username" {}
+variable "password" {}
+
+# Configure the OpenTelekomCloud Provider
+provider "opentelekomcloud" {
+  user_name = var.username
+  password = var.password
```
Comment (lego963) [inline register_host/molecule/default/files/scenarios/infrastructure/infrastructure.tf]: format code

File: register_host/molecule/default/files/scenarios/infrastructure/infrastructure.tf [hunk 8]
```diff
@@ -0,0 +1,22 @@
+terraform {
+  required_providers {
+    opentelekomcloud = ">= 1.15.0"
+  }
+}
+
+variable "username" {}
+variable "password" {}
```
Comment (lego963) [inline register_host/molecule/default/files/scenarios/infrastructure/infrastructure.tf]: may you move variables out there?

**Review by lego963 — CHANGES_REQUESTED** [review]
fix singleline comments

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by lego963 — DISMISSED** [review]

**Review by outcatcher — APPROVED** [review]

**Pull request merged**